src/components/HeaderNavMenu.jsx

```diff
@@ -30,8 +52,27 @@ function HeaderNavMenu() {
           </MuiLink>
         </Grid>
       ))}
-      <Grid item>
-        <LoginButton className={classes.menuItem} />
+      <Grid item className={classes.avatarMenu}>
+        {(loading && <null />) ||
+          (isSignedIn ? (
+            <ProfileMenu
```

Comment:
One styling I might suggest so that the avatar doesn't appear too far left is to add `margin` of value `'auto'`.

Reply:
Ok ill do that

Reply:
Hm, now the avatar is no longer next to the Links. Should this still be applied since the Avatar is now below the links and is always centered.